example/lib/task.dart

```diff
@@ -7,7 +8,10 @@ class Task {
 
   final String message;
 
-  Task(this.id, this.message);
+  // @ColumnInfo(name: '')
+  final Timestamp timestamp;
```

Comment:
I'm personally in favor of having `@embedded` here, [like in room](https://developer.android.com/reference/android/arch/persistence/room/Embedded) to be able to more finely adjust where to embed and where to type convert... what are your opinions here?

Reply:
Hello sir!

I don't have a background in Android/Java so I was not aware of the room approach.

I think that annotating the class as an embedded type is easier as we need to annotate it just once, instead of annotating every time we declare it inside an entity class. For finely adjusting, as we have just the `prefix` string, I thought we could use the `ColumnInfo name` as we already do for other variables.

If you prefer the room approach I can modify it, just tell me :)